test/support/builders/metadata/base.js

```diff
@@ -159,6 +165,11 @@ module.exports = class BaseMetadataBuilder {
     return this
   }
 
+  remote (doc /*: RemoteDoc */) /*: this */ {
```

Comment:
Should the type be `MetadataRemoteInfo`, not `RemoteDoc`?
Also what about renaming `doc` to `remoteInfos`? (`remote` would sound a bit too much like the `Remote` side to me)